doc/api/esm.md

```diff
@@ -1317,13 +1322,13 @@ _defaultEnv_ is the conditional environment name priority array,
 >       1. If _resolved_ is contained in _resolvedTarget_, then
 >          1. Return _resolved_.
 > 1. Otherwise, if _target_ is a non-null Object, then
->    1. If _target_ has an object key matching one of the names in _env_, then
->       1. Let _targetValue_ be the corresponding value of the first object key
->          of _target_ in _env_.
->       1. Let _resolved_ be the result of **PACKAGE_EXPORTS_TARGET_RESOLVE**
->          (_packageURL_, _targetValue_, _subpath_, _env_).
->       1. Assert: _resolved_ is a String.
->       1. Return _resolved_.
+>    1. For each property _p_ of _target_, in object insertion order,
```

Comment:
should this link to https://tc39.es/ecma262/#sec-ordinaryownpropertykeys ? no big deal if we do not.

Reply:
Since we explicitly ban the array indexes, and symbols do not apply to JSON sources, object insertion order seems sufficient to me.